### ISSUE TITLE
Not detecting problems

### ISSUE THREAD

**Issue by jeffjensen** [KengoTODA/findbugs-slf4j#2]
I'm not sure why it is not detecting problems.  Is there more config needed than I have?
I wrote this to test it, but no failure occurs:

```
private final Logger log = LoggerFactory.getLogger(TheClass.class);
...
    final RuntimeException ex = new RuntimeException();
    final Integer num = 4;
    final String str = "fred";
    log.debug(str + " barney={}, {}, {}, {}", num, ex);
```

I see via debug info that the plugin is loaded:

```
 [java] Loading jp.skypencil.findbugs.slf4j
 [java] Registering detector: jp.skypencil.findbugs.slf4j.WrongPlaceholderDetector
...
 [java] Adding plugin jp.skypencil.findbugs.slf4j to execution plan
```

and FB finds the class with the above snippet, so it should process it.

I only see WrongPlaceholderDetector mentioned in the output, no other skypencil detectors:

```
[java] Applying jp.skypencil.findbugs.slf4j.WrongPlaceholderDetector to (class name)
```

Here is the POM config:

```
    <plugin>
      <groupId>org.codehaus.mojo</groupId>
      <artifactId>findbugs-maven-plugin</artifactId>
      <version>${findbugsPluginVersion}</version>
      <dependencies>
        <dependency>
          <groupId>${project.groupId}</groupId>
          <artifactId>codingauditrules</artifactId>
          <version>${project.version}</version>
        </dependency>
      </dependencies>
      <configuration>
        <effort>Max</effort>
        <threshold>Low</threshold>
        <xmlOutput>true</xmlOutput>
        <excludeFilterFile>findbugs/findbugs-exclude-filters.xml</excludeFilterFile>
        <plugins>
          <plugin>
            <groupId>jp.skypencil.findbugs.slf4j</groupId>
            <artifactId>bug-pattern</artifactId>
            <version>${findbugsSlf4jRulesetVersion}</version>
          </plugin>
        </plugins>
      </configuration>
    </plugin>
```

The excludeFilterFile contents does not affect this class.

Hoping you have suggestions!
